catalog-info.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: System
+metadata:
+  name: test-frame
+  title: Test Frame
+  namespace: quality-community
+  description: |
+    Java library for easy testing of kubernetes deployments and operators using fabric8 api
+  annotations:
+    github.com/project-slug: 'skodjob/test-frame'
+  tags:
+    - java
+    - fabric8
```

Comment:
@kornys this is not part of the approved tags, can you send an MR to:

https://gitlab.cee.redhat.com/quality-innovators/qe-toolbelt/